build.sbt

```diff
@@ -1557,9 +1558,6 @@ lazy val runtime = (project in file("engine/runtime"))
     },
     Test / parallelExecution := false,
     Test / logBuffered := false,
-    Test / testOptions += Tests.Argument(
```

Comment:
This has stopped working ever since we provided a global test option for `ScalaTest`